spec/support/request_helpers.rb

```diff
@@ -4,10 +4,11 @@ module RequestHelpers
   def login(user = nil, admin: false, operator: false)
     user ||= FactoryGirl.create(:user)
 
-    case
-    when admin
+    role = { admin: admin, operator: operator }
+
+    if role[:admin]
```

Comment:
__[reek]__
* [FeatureEnvy] refers to role more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)

<!-- Comment by [SideCI](https://sideci.com) -->